host-tests/metalanguage-smoketest.lua

```diff
@@ -1,6 +1,6 @@
-local metalanguage = require "metalanguage"
-local testlanguage = require "testlanguage"
-local format = require "test-format-adapter"
+local metalanguage = require "../metalanguage"
```

Comment:
Despite these changes, this test still fails inside `testlanguage` due to a `nil` environment variable.